package.json

```diff
@@ -54,6 +54,10 @@
         "statements": 86
       }
     },
+    "moduleNameMapper": {
```

Comment:
Found the fix here: https://github.com/facebook/jest/issues/12270#issuecomment-1288163142

```
  ● Test suite failed to run

    Cannot find module '#ansi-styles' from 'node_modules/chalk/source/index.js'

      at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:491:11)
```